apps/full-site-editing/full-site-editing-plugin/new-blocks-showcase/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { addFilter } from '@wordpress/hooks';
+import { getCategories, setCategories } from '@wordpress/blocks';
+
+/**
+ * This script needs to run *before* any block is registered! This makes sure
+ * the category filter doesn't miss any blocks targeted for the "New" category.
+ */
+
+const NEW_CATEGORY = {
+	slug: 'new',
+	title: __( 'New', 'a8c' ),
+	// icon: TBD,
+};
+
+const NEW_BLOCKS = [
+	// An example content for testing purposes:
+	'a8c/blog-posts',
+	'a8c/posts-carousel',
+	'jetpack/podcast-player',
+	'jetpack/tiled-gallery',
+	'jetpack/slideshow',
+	'core/gallery',
```

Comment:
@davemart-in these are randomly selected just for testing purposes. Do we already have a list of blocks we want to include in the first release or should we wait until the PR is ready to merge and then decide?

Reply:
Let's showcase the following to start:

1. Podcast Player
2. Premium Content
3. Calendly
4. Eventbrite
5. OpenTable
6. Revue